roles/st2web/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+galaxy_info:
+  description: Install st2web
+  author: humblearner
+  company: StackStorm
+  license: Apache
+  min_ansible_version: 2.2
+  tags: st2web
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
+    - name: EL
+      versions:
+        - 6
+        - 7
+  categories:
+    - system
+dependencies:
```

Comment:
`dependencies` works OK, I see this in `ops-infra` testing nodes.
So nevermind about double checking.